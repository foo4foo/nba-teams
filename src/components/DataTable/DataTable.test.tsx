import React from 'react'
import renderer from 'react-test-renderer'
import { TableRow, TableCell } from 'grommet'
import { DataTable } from './'

describe('DataTable', () => {
  it('renders successfuly', () => {
    const tree = renderer
      .create(
        <DataTable
          headers={['1', '2', '3']}
          data={[1, 2, 3]}
          renderRow={(val) => (
            <TableRow key={val}>
              <TableCell>{val}</TableCell>
              <TableCell>{val}</TableCell>
              <TableCell>{val}</TableCell>
            </TableRow>
          )}
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
