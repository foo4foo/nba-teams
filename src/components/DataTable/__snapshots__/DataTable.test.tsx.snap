// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable renders successfuly 1`] = `
<table
  className="StyledTable-sc-1m3u5g-6 fGUmP"
>
  <thead
    className="StyledTable__StyledTableHeader-sc-1m3u5g-4 dHaGyd"
  >
    <tr
      className="StyledTable__StyledTableRow-sc-1m3u5g-2 kXgizX"
    >
      <th
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 hNAgVv"
        scope="col"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eoKPJA"
        >
          1
        </div>
      </th>
      <th
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 hNAgVv"
        scope="col"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eoKPJA"
        >
          2
        </div>
      </th>
      <th
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 hNAgVv"
        scope="col"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eoKPJA"
        >
          3
        </div>
      </th>
    </tr>
  </thead>
  <tbody
    className="StyledTable__StyledTableBody-sc-1m3u5g-3 dwInxx"
  >
    <tr
      className="StyledTable__StyledTableRow-sc-1m3u5g-2 kXgizX"
    >
      <td
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 ezepRo"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eAAZJM"
        >
          1
        </div>
      </td>
      <td
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 ezepRo"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eAAZJM"
        >
          1
        </div>
      </td>
      <td
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 ezepRo"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eAAZJM"
        >
          1
        </div>
      </td>
    </tr>
    <tr
      className="StyledTable__StyledTableRow-sc-1m3u5g-2 kXgizX"
    >
      <td
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 ezepRo"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eAAZJM"
        >
          2
        </div>
      </td>
      <td
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 ezepRo"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eAAZJM"
        >
          2
        </div>
      </td>
      <td
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 ezepRo"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eAAZJM"
        >
          2
        </div>
      </td>
    </tr>
    <tr
      className="StyledTable__StyledTableRow-sc-1m3u5g-2 kXgizX"
    >
      <td
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 ezepRo"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eAAZJM"
        >
          3
        </div>
      </td>
      <td
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 ezepRo"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eAAZJM"
        >
          3
        </div>
      </td>
      <td
        className="StyledTable__StyledTableCell-sc-1m3u5g-0 ezepRo"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 eAAZJM"
        >
          3
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;
